SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -54,13 +54,13 @@ void FRPCContainer::ProcessRPCs(const FProcessRPCDelegate& FunctionToApply)
 	}
 }
 
-bool FRPCContainer::ObjectHasRPCsQueuedOfType(const FUnrealObjectRef& TargetObjectRef, ESchemaComponentType Type) const
+bool FRPCContainer::ObjectHasRPCsQueuedOfType(const Worker_EntityId& EntityId, ESchemaComponentType Type) const
 {
 	const FRPCMap* MapOfQueues = QueuedRPCs.Find(Type);
-	if(MapOfQueues)
+	if (MapOfQueues)
```

Comment:
Driveby: Can you move the declaration inside the `if` statement.

Reply:
Updated.